doc/releasenotes/8_0_0_release_notes.md

```diff
@@ -0,0 +1,266 @@
+* Add more system settings #6486
+* Evaluate system variables #6708
+
+
+
+### VReplication
+
+* VReplication: _vt.vreplication source column VARBINARY->BLOB, allowing more tables to participate in a workflow #6421
+* VReplication and vstreamer performance and error metrics #6519, #6132
+* VReplication: Find collations for char primary keys and cast comparison during catchup #6568
+* Varchar primary keys with collations can cause vreplication to miss rows #6622
+* VReplication: Timestamp w/ Heartbeat #6635
+* VReplication: Improve few error/log messages #6669
+* VReplication: handle SET statements in the binlog stream #6772
+* VReplication: e2e Tests added to CI #6730
+* VDiff logs: additional logging around vdiff/vstreamer for observability #6684
+* VStream: experimental POC of mysql protocol #6670
+* Materialize: Support rollups using a literal value as a column #6733
+* Materialization: Ignore non-participating tables that are renamed/deleted #6778
+* Materialize: add ability for maintaining accurate table-level aggregates #6767
+* Schema Tracker: Turn off schema tracker by default #6712 (breaking change)
+* Workflow: Add List-All Command #6533
+* Workflow: List -> Show, and Expand Metadata #6544
+* Refactor: Cleanup Noisy Workflow Logging #6744
+* Schema tracker: Change tracker to use the allprivs user instead of dba #6720
+* vtctl: Adding VExec and Workflow commands #6410
+* vtctl Workflow cmd: don't expect all shards to have workflow streams #6576
+* Reshard: Sort table definitions before comparing them #6765 #6738
+* Expose a shard range calculator as a vtctl command GenerateShardRanges #6751 #6752
+* VStream: Field event now has all column attributes #6525
+* VStream events do not populate field "Flags" on FIELD events #5914
+* RFC: VReplication based SplitClone #4604
+* Experimental: Mysql Protocol support for VStream #6675
+* VReplication: Add metrics around vstreamer and vreplication metrics #6787
+* Fix issues with missing tables in vstreams #6789
+* Add validations and logs to fix common issues faced during MoveTables/Reshard #6814
+* Allow multiple blacklists for master #6816
+
+### VTtablet
+* VTtablet throttling #6668
+https://vitess.io/docs/reference/features/tablet-throttler/
+* TabletManager: publish displayState #6648
+* TabletManager: call setReadOnly inside ChangeTabletType #6762
+* TabletManager: change how SetReadOnly is called to avoid errors from externally managed tablets #6786
+* TabletManager: call SetReadOnly inside ChangeTabletType #6804
+* vttablet: Open and Close healthStreamer #6515
+* vttablet: create database if not present #6490
+* Restore: do not change tablet type to RESTORE if not actually performing a restore #6679
+* Restore: checkNoDB should not require tables to be present #6695
+* Heartbeat: Additional metrics/stats (lag histogram) #6528 #6634
+* VTtablet throttler feature flag: -enable-lag-throttler #6815
+* VTtablet two-phase commit design doc #6498
+
+### VTorc - Orchestrator
```

Comment:
We can summarize the bullets below into: "Experimental version of Vitess-native Orchestrator 'vtorc' is ready for users to try out.

Reply:
Done